SyncTelemetry/SyncTelemetry.swift

```diff
@@ -1,130 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Common
-import Foundation
-import SwiftyJSON
-import Shared
-
-private let ServerURL = "https://incoming.telemetry.mozilla.org".asURL!
-private let AppName = "Fennec"
-
-public enum TelemetryDocType: String {
-    case core
-    case sync
-}
```

Comment:
@lougeniaC64 replaced by glean?

Reply:
@nbhasin2 Yep! The sync telemetry logic we use is in `SyncTelemetryUtils.swift`.